docs/en/framework/real-time/signalr.md

```diff
@@ -55,11 +55,9 @@ Client side installation depends on your UI framework / client type.
 Run the following command in the root folder of your web project:
 
 ```bash
-yarn add @abp/signalr
+npm install @abp/signalr
```

Comment:
Is there an ABP CLI command that installs ABP NPM packages?

Reply:
The `add-package` command of `ABP CLI` is intended to add NuGet and Angular packages.

Reply:
https://github.com/abpframework/abp/pull/23136